src/ClientHandle.cpp

```diff
@@ -1924,7 +1924,7 @@ void cClientHandle::Tick(float a_Dt)
 		{
 			m_PingID++;
 			m_PingStartTime = std::chrono::steady_clock::now();
-			m_Protocol->SendKeepAlive(m_PingID);
+			m_Protocol->SendKeepAlive(static_cast<UInt32>(m_PingID));
```

Comment:
As it doesn't matter what the value is, it would be easier to change m_PingID than go through and change cProtocol::SendKeepAlive() and any overloaded methods, along with cByteBuffer::WriteVarInt32().